package.json

```diff
@@ -2,10 +2,10 @@
   "name": "uk-coronavirus-data-alerts",
   "version": "0.0.1",
   "dependencies": {
-    "@guardian/cdk": "^15.1.0",
+    "@guardian/cdk": "guardian/cdk#make-it-possible-to-define-multiple-lambdas-per-stack",
```

Comment:
TODO: publish the cdk library with [our fix](https://github.com/guardian/cdk/pull/781)

Reply:
Feel free to merge https://github.com/guardian/cdk/pull/781 whenever you're ready - it should be released automatically